<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>extrapolate_Dialog</class>
 <widget class="QDialog" name="extrapolate_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extrapolate Log</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Well</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="well_comboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="log_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Density Log</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>440</y>
        <width>141</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Horizons</string>
      </property>
      <widget class="QPushButton" name="plot_horizon_Button">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>65</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
      <widget class="QListWidget" name="horizon_listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>171</height>
        </rect>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>141</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Extrapolate</string>
      </property>
      <widget class="QPushButton" name="fit_Button">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>65</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Fit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Interval for fit</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="start_lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>60</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Start Depth</string>
       </property>
       <property name="text">
        <string>1500</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="end_lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>60</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>End Depth</string>
       </property>
       <property name="text">
        <string>4000</string>
       </property>
      </widget>
      <widget class="QPushButton" name="extrapolate_Button">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>160</y>
         <width>65</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Extrapolate</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="a_lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>a</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="b_lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>b</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="draw_Button">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>65</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Draw</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clear_Button">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>65</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>a:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>b:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>start:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>end:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>smoothed Density</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="sm_log_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Save extrapoalted log</string>
      </property>
      <widget class="QPushButton" name="save_Button">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>65</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>_extra</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
